src/Model/CustomerAccount.java

```diff
@@ -0,0 +1,123 @@
+package Model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import API.CartService;
+import API.UserService;
+import Model.paymentMethods.Cash;
+import Model.paymentMethods.CreditCard;
+import Model.paymentMethods.FinancialPayment;
+import Model.paymentMethods.PaypalAccount;
+
+
+
+public class CustomerAccount implements CustomerService {
```

Comment:
The idea here is to provide e CustomerService that will handle a Customer or a CustomerAccount.